src/pedal.cpp

```diff
@@ -46,4 +50,41 @@ void Pedal::Reset()
 // Pedal functor methods
 //----------------------------------------------------------------------------
 
+int Pedal::GenerateMIDI(FunctorParams * functorParams)
+{
+    GenerateMIDIParams *params = dynamic_cast<GenerateMIDIParams *>(functorParams);
+    assert(params);
+
+    if (!HasDir()) return FUNCTOR_CONTINUE;
```

Comment:
We should return `FUNCTOR_SIBLINGS` or `FUNCTOR_CONTINUE` consistently even if will  make no difference